core/src/main/java/org/stellar/anchor/asset/DefaultAssetService.java

```diff
@@ -95,7 +132,9 @@ private static DefaultAssetService createDASFromMap(Map<String, List<Object>> ma
         throw new InvalidConfigException(String.format("Invalid asset: " + id));
       }
     }
-    AssetValidator.validate(das);
+    if (validate) {
```

Comment:
What’s the case where we wouldn’t want to validate?

Reply:
When loading the prototype yaml file (`anchor-asset-default-values.yaml`), the validation is off.